docs/dotnet.rst

```diff
@@ -0,0 +1,152 @@
+  - The application's source code is hosted on a Git server such as GitHub, GitLab, or Bitbucket
+  - A Git commit with a `Tag <https://git-scm.com/book/en/v2/Git-Basics-Tagging>`__ exists on the repository to use for the build. This usually represents the version number of the application (e.g. v1.0, v1.1, etc.).
+
+Steps for Packaging
+~~~~~~~~~~~~~~~~~~~
+
+Installing dependencies
+^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Install Flatpak using the method provided for your distribution
+   `Flatpak - Quick Setup <https://flatpak.org/setup/>`__
+
+2. Install Flatpak Builder as both a flatpak package and a distribution package (apt / dnf)
+
+  .. code-block:: shell
+
+        flatpak install org.flatpak.Builder
+        sudo apt install flatpak-builder
+
+3. Install the FreeDesktop SDKs (When prompted for a version, select the
+   latest available version)
+
+  .. code-block:: shell
+
+        flatpak install flathub org.freedesktop.Platform org.freedesktop.Sdk
+
+4. Install the Flatpak SDK extension for your dotnet version
+   (e.g. `dotnet6 <https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet6>`__, `dotnet7 <https://github.com/flathub/org.freedesktop.Sdk.Extension.dotnet7>`__)
+
+  .. code-block:: shell
+
+        flatpak install org.freedesktop.Sdk.Extension.dotnet6
+
+
+Creating the Flatpak
+^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+  Here is a brief description of the placeholders in the below example:
+
+  - ``<namespace>``: The prefix that goes before the name of your Flatpak, which uniquely identifies the author. See `"Conventions - Application IDs" section <https://docs.flatpak.org/en/latest/conventions.html#application-ids>`__ to find the appropriate value for your use case.
+  - ``<app-name>``: The name of your application to use for Flatpak, usually somewhat similar to the project name inside the solution
+  - ``<project-name>``: The name of your ``.csproj`` file
+  - ``<user-name>``: The username you use in your Git server (GitHub, GitLab, Bitbucket)
+  - ``<release-number>``: The numbered release to build from, in the form of a Git `Tag <https://git-scm.com/book/en/v2/Git-Basics-Tagging>`__.
+
+5.  Create a new folder somewhere different from your existing project
+
+
+6.  Create a YAML file titled
+    ``<namespace>.<app-name>.yaml`` with the following
+    example template, replacing the placeholders with the appropriate
+    information: \
+
+  .. code-block:: yaml
+
+      app-id: <namespace>.<app-name>
+      runtime: org.freedesktop.Platform
+      runtime-version: '22.08'
+      sdk: org.freedesktop.Sdk
+      sdk-extensions:
+        - org.freedesktop.Sdk.Extension.dotnet6
+      build-options:
+        prepend-path: "/usr/lib/sdk/dotnet6/bin"
+        append-ld-library-path: "/usr/lib/sdk/dotnet6/lib"
+        env:
+          PKG_CONFIG_PATH: "/app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet6/lib/pkgconfig"
```

Comment:
you should use `append-pkg-config-path` instead (or `prepend-pkg-config-path`)

Reply:
Just updated this one as well to `prepend-pkg-config-path`